### DISCUSSION TITLE
Using a regular ubi8 image instead of ubi8-minimal

### DISCUSSION THREAD

**Discussion by marcrubia** [quarkusio/quarkus#38526] [Q&A]
Hi,

Quarkus recommended image for Java 17 (ubi8/openjdk-17) extends from the ubi8-minimal, which have a lot of restrictions in terms of package manager (comes with microdnf instead of yum). Tried to install yum using microdnf because we need whois installed in our Quarkus image, but unfortunately it doesn't work well as it doesn't have many repositories, only finds jwhois and jwhois doesn't work at all.

Anyone had luck using a full redhat distribution that comes with yum? We are migrating from the old centos7 images after moving from Jersey+Java8 to Quarkus+Java17 and this is our main pain.

[the remark this answers is not in the document]
**Reply by maxandersen** [reply]
The full red hat images don't have a license that allow free (as in beer and speech) distribution so that is not an option Afaics. 

Also not using ubi minimal would add size overheads too. 

Is there something preventing you from using the red hat full images in your own container builds which then allows you to use all extra kind of packages that you are after ?

[the remark this answers is not in the document]
**Reply by marcrubia** [reply]
Thank you for your answer @maxandersen 
I understand the license+size concerns. Make sense from Quarkus POV. 
Answering your question, haven't been able to package full ubi + openjdk 17 and run Quarkus in a few tests I did, a regular entrypoint running the jar didn't work. I reviewed the base ubi8/openjdk-17 Dockerfile which can be found [here](https://catalog.redhat.com/software/containers/ubi8/openjdk-17/618bdbf34ae3739687568813?architecture=amd64&image=65a905ecbccdec7b2b87c6eb&container-tabs=dockerfile) and it comes with a lot of jboss magic that after checking their repos require extra effort to modify and build (starting with Cekit). 

So I guess we'll search for a workaround or a way to skip the whois command, as it seems cronie and untar which are the other required packages are working OK with ubi-minimal

Thanks again.
